src/core/ui/components/sign-in/header.module.css

```diff
@@ -0,0 +1,39 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 28px;
+  padding-top: 32px;
```

Comment:
Out of curiosity are we using pixels everywhere or are we using rem anywhere?

Reply:
Most are px but there are some rem here and there for margins:
![image](https://user-images.githubusercontent.com/17853818/211658029-56a6aef7-ef03-4ca4-b684-0b4d5219725a.png)
![image](https://user-images.githubusercontent.com/17853818/211658223-6cacc2d3-f647-4b28-9890-1fd5d4cf274d.png)
![image](https://user-images.githubusercontent.com/17853818/211658409-e8aa006a-89fe-41c2-a173-7e4ab6c4139b.png)
 
![image](https://user-images.githubusercontent.com/17853818/211659409-a2bf48d6-3808-4140-80cb-a6e09e5b67c2.png)

I would change to always use relative measurements.

Reply:
Seems reasonable -- let's file this in another ticket that we can address in the future? Or do you think we should do it now?

Reply:
I would prefer to tackle it at once, to make sure we do not leave some px's behind. What about doing so in any of this stories:
- https://linear.app/slashauth/issue/SLA-1845/sr-theme-and-customization-of-login-component (Im already doing the theming in each of the components)
- https://linear.app/slashauth/issue/SLA-1927/sr-singin-component-release-redesign

Reply:
I decided to add it to the signin component release